apps/extension/src/ui/apps/dashboard/routes/Networks/Pill.tsx

```diff
@@ -0,0 +1,24 @@
+import { classNames } from "@talismn/util"
+import { ReactNode } from "react"
+import { useTranslation } from "react-i18next"
+
+export const Pill = ({ className, children }: { className?: string; children?: ReactNode }) => (
```

Comment:
might not want to export this, name being generic it could be pollute intellisense